index.css

```
@@ -0,0 +1,186 @@
+body {
+    background-color: #f2f2f2;
+    color: #333;
+    font-weight: 400;
+    line-height: 1.3;
+    margin: 32px;
+    font-family: Arial, Arial, Helvetica, sans-serif;
+}
+
+img {
+    max-width: 100%;
+}
+
+.nav {
+    box-sizing: border;
+    padding: 15px;
+}
+
+.topnav {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+}
+
+.topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+
+.topnav a:hover {
+    background-color: #ddd;
+    color: black;
+}
+
+.u-list-reset {
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+}
+
+.title {
+    box-sizing: border;
+    padding: 10px;
+    margin-top: -30px;
+    margin-bottom: -10px;
+    margin-left: 5px;
+    margin-right: 5px;
+}
+
+.big1 {
+    font-size: 50px;
+    background-color: #333;
+    width: 18%;
+    color: white;
+}
+
+.big2 {
+    font-size: 50px;
+    background-color: #333;
+    width: 15%;
+    color: white;
+}
+
+.h1title {
+    padding-bottom: 20px;
+    border-bottom: 3px solid #22333B;
+}
+
+.strong a {
+    font-family: Arial, Helvetica, sans-serif;
+    color: #333;
+    text-decoration: underline overline;
+}
+
+.author-img {
+    border-radius: 50%;
+    height: auto;
+    margin-right: auto;
+    margin-bottom: 12px;
+    margin-left: auto;
```

The `margin-left` and `margin-right` rules here aren't having any effect:
```suggestion
    margin-bottom: 12px;
```